src/nnbench/reporter/file.py

```diff
@@ -0,0 +1,205 @@
+from __future__ import annotations
+
+import os
+from typing import Any, List
+
+from nnbench.reporter.base import BenchmarkReporter
+from nnbench.types import BenchmarkRecord
+
+
+class Parser:
```

Comment:
Just for clarification: We're not actually parsing things when we are reading files, that is the job of the respective modules (`json|yaml|toml`).